src/main/java/org/motechproject/mots/service/ModuleAssignmentService.java

```diff
@@ -61,7 +61,8 @@ public void assignModules(AssignedModules assignedModules) {
       ivrService.addSubscriberToGroups(ivrId,
           getIvrGroupsFromModules(existingAssignedModules.getModules()));
     } catch (IvrException ex) {
-      String message = "Could assign module for CHW, because of IVR module assignment error";
+      String message = "Could assign module for CHW, because of IVR module assignment error. "
+          + ex.getClearVotoInfo();
```

Comment:
I would just add a new line before ex.getClearVotoInfo() to make it more readable